bin.js

```diff
@@ -50,14 +50,24 @@ function degit(dir) {
 
 async function patch(dir) {
   try {
-    if (!fs.existsSync(join(dir, 'src/server.js'))) {
+    const serverFile = fs.existsSync(join(dir, 'src/server.js'))
+      ? 'src/server.js'
+      : fs.existsSync(join(dir, 'src/server.ts'))
+      ? 'src/server.ts'
+      : ''
+
+    if (!serverFile) {
       throw new Error(
-        'expected src/server.js to exist, is this really a Sapper project?'
+        `Expected src/server.{js|ts} to exist, is this really a Sapper project?`
       )
     }
-    const server = fs.readFileSync(join(dir, 'src/server.js'), 'utf8')
-    fs.writeFileSync(join(dir, 'src/server.js'), patchServer(server))
-    console.log(green('Patched'), 'src/server.js')
+
+    const server = fs.readFileSync(join(dir, `${serverFile}`), 'utf8')
```

Comment:
This could also be improved for both speed and readability

```js
const path = join(dir, serverFile)
const server = fs.readFileSync(path, 'utf8')

fs.writeFileSync(
      path,
      patchServer(server, serverFile)
    )
    console.log(green('Patched'), serverFile)
```

Reply:
Saw that too ^^